\documentclass[titlepage,11pt, oneside]{article}   	% use "amsart" instead of "article" for AMSLaTeX format
\usepackage{geometry}                		% See geometry.pdf to learn the layout options. There are lots.
\usepackage[utf8]{inputenc}
\geometry{letterpaper}                   		% ... or a4paper or a5paper or ... 
%\geometry{landscape}                		% Activate for rotated page geometry
%\usepackage[parfill]{parskip}    		% Activate to begin paragraphs with an empty line rather than an indent
\usepackage{graphicx}				% Use pdf, png, jpg, or eps§ with pdflatex; use eps in DVI mode
								% TeX will automatically convert eps --> pdf in pdflatex		
\usepackage{amssymb}
\usepackage{authblk}
\usepackage{indentfirst}
\usepackage{gensymb}
\usepackage{hyperref}
%SetFonts


\title{\textbf{Complete chloroplast genome sequence of a white spruce (\textit{Picea glauca}) genotype from eastern Canada}}
\author{TBD}
\date{January 24, 2019}					% Activate to display a given date or no date

\begin{document}
\maketitle

\section*{Genome Announcement}

The \textit{P. glauca} isolate WS77111 sample was collected from Otonabee-South Monaghan (44\degree 19' 48" N 78\degree 9' 0" W; elevation of 250 m) in southern Ontario. The tissue sample collected was from the needles of the white spruce tree, supplied by John MacKay of Universite Laval. These tissue samples were then sequenced at the British Columbia Cancer’s Genome Sciences Centre as an objective of the SpruceUp and SMarTForests project.
\newline
\par
Genomic DNA libraries were constructed according to British Columbia Cancer’s Genome Sciences Centre plate-based and paired-end library protocols on a Microlab NIMBUS liquid handling robot (Hamilton, USA). Briefly, 1 $\mu$g of high molecular weight genomic DNA was sonicated (Covaris LE220) in 62.5 $\mu$L volume to 400 bp. Sonicated DNA was purified with PCRClean DX magnetic beads (Aline Biosciences). The DNA fragments were end-repaired, phosphorylated and bead purified in preparation for A-tailing using a custom NEB Paired-End Sample Prep Premix Kit (New England Biolabs).  Illumina sequencing adapters were ligated overnight at 16oC and adapter ligated products bead purified and enriched with 6 cycles of PCR using primers containing a hexamer index that enables library pooling. Pooled libraries were sequenced with paired-end 250 bp reads on an Illumina HiSeq2500 instrument in rapid mode.
\newline
\par
To assemble the complete chloroplast genome, these reads were subsampled in the sizes of read pairs (in millions). Each subset was then assembled using ABySS v2.1.0. Then, each assembly was filtered for the chloroplast genome by alignment to the reference chloroplast genome, the \textit{Picea glauca} isolate PG29 complete chloroplast genome (Genbank KT634228), where only contigs greater or equal to 500 bp that aligned were kept, using BWA v0.7.17. The resulting contigs were pieces of WS77111 chloroplast, assessed using QUAST v5.0.2 and the reference chloroplast genome. Of all the subsets, the 1.5M subset had the least number of misassemblies, with its chloroplast genome in 14 pieces, while the 6M subset had the least number of pieces at 6, but contained 1 misassembly. For this reason, the 1.5M, 6M and the 3M subset (subset in between) were chosen to advance for an ABySS parameter sweep. Of these consequent assemblies, two were chosen to advance to the next step: 1.5M and 6M assembly. LINKS v1.8.5 revealed that the 1.5M and 6M subset contained the fewest pieces and misassemblies. After LINKS, both assemblies had combined the chloroplast genome into one single piece (the largest resulting contig). QUAST analysis showed that the 1.5M subset contained 12 gaps, and the 6M one contained 5 gaps. After using Sealer (part of ABySS) to close the gaps, 1.5M subset had the fewest remaining gaps and thereby determined to be the best assembly. This assembly was then modified to have the same start and end as the reference chloroplast genome using BLAST v.2.7.1. Finally, Pilon v1.22 was run to polish the final assembly.
\newline
\par
The WS77111 chloroplast genome is 123,421bp in length, with a GC content of 38.74\%. It has a total of 114 genes: 74 protein-coding genes, 36 tRNA-coding genes, 4 rRNA-coding genes. This chloroplast genome was annotated using GeSeq, with all available \textit{Picea} NCBI RefSeq chloroplast genomes as reference genomes. Five genes required manual annotation: \textit{rps12}, \textit{petB}, \textit{petD}, \textit{rpl16}, and \textit{psbZ}. OGDRAW v1.2 was draw to visualize the circular chloroplast genome map. Adding the chloroplast genome of WS77111 to the resource pool of available \textit{Picea} chloroplasts provides valuable insight to the phylogeny and evolution of these spruce trees.
\newline
\newline
\textbf{Accession number(s).} The complete chloroplast genome sequence of \textit{Picea glauca}, isolate WS77111 can be found in NCBI Genbank under the accession number MK174379. The tissue samples used can be found under BioSample: SAMN02736787 and BioProject: PRJNA242552. The NCBI accession numbers of the other \textit{Picea} chloroplasts used in annotation are as follows: \textit{Picea abies} (NC\_021456), \textit{Picea asperata} (NC\_032367), \textit{Picea glauca} (NC\_028594), \textit{Picea morrisonicola} (NC\_016069), and \textit{Picea sitchensis }(NC\_011152).

\end{document}  